configs/efficientdet/retinanet_effd0_bifpn_1x_coco.py

```diff
@@ -103,7 +103,7 @@
             dict(type='Resize', keep_ratio=True),
             dict(type='RandomFlip'),
             dict(type='Normalize', **img_norm_cfg),
-            dict(type='Pad', size_divisor=32),
+            dict(type='Pad', size_divisor=128),
```

Comment:
what about training?

Reply:
No problems with current config, 'cause 512x512 image crops are used during training. Agree that it's better to change padding there as well, though.